.denov

```diff
@@ -1 +0,0 @@
-v1.2.0
```

Comment:
`.denov` is used by [dink-bot](https://github.com/keroxp/dink-bot) through `.github/workflows/bump.yml`.
Could you revert this for now?

Reply:
I see, I'll do that